app/views/standard/confirmDeleteAdditionalMetadata.scala.html

```diff
@@ -31,3 +31,11 @@ <h1 class="govuk-heading-xl">
             </div>
         }
     }
+
+    @displayWarningMessage(hasEnteredMetadata: Boolean) = {
+        @if(hasEnteredMetadata) {
+            @warningMessage(Messages(s"deleteAdditionalMetadata.${metadataType}DeletionWarningMessage"))
+        } else {
+            @warningMessage(Messages(s"deleteAdditionalMetadata.noMetadataWarningMessage", routes.AdditionalMetadataNavigationController.getAllFiles(consignmentId, metadataType, expanded = None), "file selection page"))
+        }
+    }
```

Comment:
Can you please update the tests to verify the warning message as well?

Reply:
As suggested I've updated to tests to verify the warning messages